src/js/models/search/SearchAwardsOperation.js

```diff
@@ -0,0 +1,241 @@
+        }
+
+        // Add Agencies
+        if (this.fundingAgencies.length > 0 || this.awardingAgencies.length > 0) {
+            const agencies = [];
+
+            // Funding Agencies are toptier-only
+            this.fundingAgencies.forEach((agencyArray) => {
+                agencies.push({
+                    [agencyKeys.type]: 'funding',
+                    [agencyKeys.tier]: 'toptier',
+                    [agencyKeys.name]: agencyArray.toptier_agency.name
+                });
+            });
+
+            // Awarding Agencies can be both toptier and subtier
+            this.awardingAgencies.forEach((agencyArray) => {
+                const agencyName = agencyArray.agencyType === 'toptier'
+                    ? agencyArray.toptier_agency.name
+                    : agencyArray.subtier_agency.name;
+
+                agencies.push({
+                    [agencyKeys.type]: 'awarding',
+                    [agencyKeys.tier]: agencyArray.agencyType,
+                    [agencyKeys.name]: agencyName
+                });
+            });
+
+            filters[rootKeys.agencies] = agencies;
+        }
+
+        // Add Recipients, Recipient Scope, Recipient Locations, and Recipient Types
+        if (this.selectedRecipients.length > 0) {
+            filters[rootKeys.recipients] = this.selectedRecipients.map(
+                (recipient) => recipient.legal_entity_id);
+        }
+
+        if (this.recipientDomesticForeign !== '' && this.recipientDomesticForeign !== 'all') {
+            filters[rootKeys.recipientLocationScope] = this.recipientDomesticForeign;
+        }
+
+        if (this.selectedRecipientLocations.length > 0) {
+            let locationSet = [];
+            this.selectedRecipientLocations.forEach((location) => {
+                locationSet = concat(locationSet, location.matched_ids);
+            });
+
+            filters[rootKeys.recipientLocation] = locationSet;
+        }
+
+        if (this.recipientType.length > 0) {
+            filters[rootKeys.recipientType] = this.recipientType;
+        }
+
+        // Add Locations
+        if (this.selectedLocations.length > 0) {
+            let locationSet = [];
+            this.selectedLocations.forEach((location) => {
+                locationSet = concat(locationSet, location.matched_ids);
+            });
+
+            filters[rootKeys.placeOfPerformance] = locationSet;
+        }
+
+        if (this.locationDomesticForeign !== '' && this.locationDomesticForeign !== 'all') {
+            filters[rootKeys.placeOfPerformanceScope] = this.locationDomesticForeign;
+        }
+
+        // Add Award Amounts
+        if (this.awardAmounts.length > 0) {
+            const amounts = [];
+
+            this.awardAmounts.forEach((awardAmount) => {
+                const amount = {};
+
+                if (awardAmount[0] !== 0) {
+                    amount[awardAmountKeys.min] = awardAmount[0];
+                }
+
+                if (awardAmount[1] !== 0) {
```

Comment:
The backend is asking **only** for defined elements, which is why this differs a bit from the previous implementation. I can add the check to delete the max number if the min number is greater than it, similar to before.